contrib/alacritty/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "alacritty"
+pkgver = "0.12.2"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "cmake", "pkgconf", "python", "ncurses"]
+makedepends = ["fontconfig-devel", "freetype-devel", "libxcb-devel", "rust"]
+depends = [f"alacritty-terminfo={pkgver}-r{pkgrel}"]
+pkgdesc = "Cross-platform, GPU-accelerated terminal emulator"
+maintainer = "nbfritch <nbfritch@gmail.com>"
+license = "MIT OR Apache-2.0"
+url = "https://github.com/alacritty/alacritty"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "285b44d2d2e83118ab7fe61d575811bb85d5e21147808171bd0e583e9e794748"
+
+
+def do_install(self):
+    self.cargo.install(wrksrc="alacritty")
```

Comment:
you can probably remove this and turn the `do_install` into a `post_install`

Reply:
install --path=. doesn't work in workspaces, you need either this wrksrc override or the proposed change in https://github.com/chimera-linux/cports/pull/259 .

Reply:
alright, let's go with this for now